src/org/exist/collections/CollectionCache.java

```diff
@@ -38,13 +39,13 @@
  * 
  * @author wolf
  */
-public class CollectionCache extends LRUCache {
+public class CollectionCache extends LRUCache implements BrokerPoolService {
 
     private Object2LongHashMap names;
     private BrokerPool pool;
 
     public CollectionCache(BrokerPool pool, int blockBuffers, double growthThreshold) {
-        super(blockBuffers, 2.0, 0.000001, CacheManager.DATA_CACHE);
+        super(blockBuffers, 2.0, growthThreshold, CacheManager.DATA_CACHE);
```

Comment:
@shabanovd Yes that is correct, however from how the code was written, I suspect the growthThreshold was meant to be `0.0001`: https://github.com/eXist-db/exist/blob/develop/src/org/exist/storage/BrokerPool.java#L840

Do you agree?